usaspending_api/settings.py

```diff
@@ -173,6 +173,7 @@ def _configure_database_connection(environment_variable):
     default_options = {"options": "-c statement_timeout={0}".format(DEFAULT_DB_TIMEOUT_IN_SECONDS * 1000)}
     config = dj_database_url.parse(os.environ.get(environment_variable), conn_max_age=CONNECTION_MAX_SECONDS)
     config["OPTIONS"] = {**config.setdefault("OPTIONS", {}), **default_options}
+    config["TEST"] = {"SERIALIZE": False}
```

Comment:
The whole `TEST` section is specifically designed to override database settings during tests.  I would hope it wouldn't cause any issues.  This specific setting allows you to emulate rollbacks for test databases that don't support rollbacks.  Since PostgreSQL supports rollbacks, this settings _SHOULD BE_ safe.

Reply:
What Kirk said. This was an attempt at a performance optimization, since it is true by default. It basically creates a giant dump of the DB (with data I think) and saves it in the `connection` object to give it something to reinstate to simulate a rollback. Turns out it didn't have much performance impact, but a better config state since none of our tests rely on this.